instagram/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './App.css';
+import Search from './components/SearchBar/Search';
+import Posts from './components/PostContainer/Posts';
+import data from './dummy-data'
+
+class App extends Component {
+
+  state = {
+    data: data
+  }
+  render() {
+    return (
+      <div className="App">
+        <Search />
+        <Posts data={this.state.data}/>
```

Comment:
If you have state just for the sake of passing it to a component you can pass it to the component directly. For example

`<Posts data={data}/>` and the `data` inside of the `{}` is from `./dummy-data`

Reply:
I did not know that! Awesome! Thanks